java/Fizzbuzz.java

```diff
@@ -0,0 +1,23 @@
+import java.util.*;
+public class Fizzbuzz{
+	public static void main(String[] args){
+		System.out.println("Enter the number:");
+		int n=100;
+		for(int i=1;i<=n;i++){
+			if(i%5 == 0){
```

Comment:
Wrong logic.
For 15, it will print Buzz as the first condition will satisfy and other else if conditions won't be reached. You should have the last else if with the check for 3 and 5 to be the very first check.